check-ecobalyse-data-sync.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+set -euo pipefail
+
+# Directory where files will be downloaded
+TEMP_DIR=$(mktemp -d)
+ECOBALYSE_DATA_BRANCH="${ECOBALYSE_DATA_BRANCH:-main}"
+RAW_GITHUB_URL="https://raw.githubusercontent.com/MTES-MCT/ecobalyse-data/${ECOBALYSE_DATA_BRANCH}"
```

Comment:
Nit: you may want to make the URL configurable with an env variable as you did with the branch name here

Reply:
done